web/packages/teleport/src/components/ReAuthenticate/useReAuthenticate.ts

```diff
@@ -90,4 +113,23 @@ export type Props = {
   actionText?: string;
 };
 
+// MfaResponseProps defines a function
+// that accepts a MFA response. No
+// authentication has been done at this point.
+type MfaResponseProps = BaseProps & {
+  onMfaResponse(res: MfaAuthnResponse): void;
+  onAuthenticated?: never;
+};
+
+// Default defines a function that
+// accepts a privilegeTokenId that is only
+// obtained after MFA response has been
+// validated.
+type Default = BaseProps & {
```

Comment:
Can we name this something other than `Default`?

Reply:
asked discussed in call, renamed to `DefaultProps`